wzdx/geotab_heavy_fleet.json

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
Where are we using this one? Is it sample data we want to include in a directory somewhere?

Reply:
removed